saleor/graphql/middleware.py

```diff
@@ -37,8 +37,14 @@ def user():
 class ChannelMiddleware:
     def resolve(self, next, root, info, **kwargs):
         request = info.context
-
-        channel_slug = kwargs.get("channel")
+        request_input = kwargs.get("input", {})
```

Comment:
This is magic. Some mutations will have `input` param, some others will not. Second thing - I'm afraid it will be really hard to guess what value is actually stored in the request if you're using it somewhere in the code. We need to discuss why we need these changes and if can be done in some more explicit and easy to follow way.

Reply:
Maby if these changes are required only for CreateCheckout mutation we shouldn't put in inside mutation?